src/actions/general.js

```diff
@@ -94,8 +94,8 @@ export function setServerVersion(serverVersion) {
 
 export function setStoreFromLocalData(data) {
     return async (dispatch, getState) => {
-        Client.setToken(data.token);
-        Client.setUrl(data.url);
+        Client4.setToken(data.token);
+        Client4.setUrl(data.url);
```

Comment:
Im afraid that if we do this the `Client` won't have the token or the url set? can we keep it in both for now?